doc/ref_impl/cntt-ri2/chapters/chapter04.md

```diff
@@ -7,27 +7,53 @@
 * [4.1 Introduction](#4.1)
 * [4.2 Prerequisites](#4.2)
 * [4.3 Installation of the Reference Implementation](#4.3)
-  * [4.3.1 Placeholder for Installer-A](#4.3.1)
-  * [4.3.2 Placeholder for Installer-B](#4.3.2)
+  * [4.3.1 Node Provisioning](#4.3.1)
+  * [4.3.2 Software Provisioning](#4.3.2)
 * [4.4 Validation of the Reference Implementation](#4.4)
 * [4.5 Automation Tooling](#4.5)
 
 <a name="4.1"></a>
 ## 4.1 Introduction
 
-> Describe the purpose of the chapter - e.g. to describe in detail the stages to run through for anyone to be able to deploy the RI in their lab.
+This chapter describes the steps to install Kubernetes based Reference Implementation (RI-2). The entire installation is divided into two stages - Node provisioning & Software provisioning. The former stage can be skipped when using Bare Metal Providers like Packet, etc. for node provisioning.
 
 <a name="4.2"></a>
 ## 4.2 Prerequisites
 
-> Describe the prerequisites needed in order to install the RI - e.g. all requirements in chapter03.md are met.
-
+Before attempting the installation, please ensure that all requirements described in chapter03.md are met.
+(More details to follow)
 
 <a name="4.3"></a>
 ## 4.3 Installation of the Reference Implementation
 
-> Describe the tasks required in order to install the RI.
+<a name="4.3.1"></a>
+### 4.3.1 Node Provisioning
```

Comment:
@pgoyal01 made a good comment on my PR regarding the terminology here. "node" can be understood as a Kubernetes node while in fact it only refers to the host OS here. So Host Provisioning might be a better term. Also applicable to all other occurences of "node" here. 
```suggestion
### 4.3.1 Host Provisioning
```